### PR TITLE
OCLOMRS-729:Add ranges, units, and decimal precision to the UI when creating a new concept where datatype is numeric

### PR DESCRIPTION
**Pull request by reagan-meant**
Added ranges, units, and decimal precision to the UI when creating a new concept,viewing concept and editing a concept where datatype is numeric.
https://issues.openmrs.org/browse/OCLOMRS-729

### CODE REVIEW

**Comment by dkayiwa** [comment]
@reagan-meant did you see the travis build failure?

**Comment by reagan-meant** [comment]
Yes @dkayiwa I will respond to them hopefully in the coming week.... Currently am unable to work on this

File: src/tests/dictionaryConcepts/components/CreateConceptForm.test.jsx [hunk 1]
```diff
@@ -173,4 +191,62 @@ describe('Test suite for CreateConceptForm', () => {
...
+        numericEnabled: true,
+      },
+      allSources: [mockSource],
+    };
+    const wrapper = shallow(<CreateConceptForm {...props} />);
+    expect(wrapper.find('NumericPrecision')).toHaveLength(1);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render form for numericPrecision when dataType is numeric', () => {
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/components/CreateConceptForm.test.jsx]: What is the difference between this test and the one above?

Comment (reagan-meant) [reply src/tests/dictionaryConcepts/components/CreateConceptForm.test.jsx]: I will clarify the test name...This is intended to test when your editing a concept and not creating......(line 212 and line 241)

**Pull request closed without merging**